windows/ReactTestApp/MainPage.h

```diff
@@ -1,33 +1,24 @@
 #pragma once
 
 #include "MainPage.g.h"
+#include "ReactInstance.h"
 
 namespace winrt::ReactTestApp::implementation
 {
-    enum class JSBundleSource {
-        DevServer,
-        Embedded,
-    };
 
     struct MainPage : MainPageT<MainPage> {
     public:
         MainPage();
-
         void LoadFromJSBundle(Windows::Foundation::IInspectable const &,
                               Windows::UI::Xaml::RoutedEventArgs);
         void LoadFromDevServer(Windows::Foundation::IInspectable const &,
                                Windows::UI::Xaml::RoutedEventArgs);
 
     private:
-        winrt::Microsoft::ReactNative::ReactNativeHost m_reactNativeHost;
-        std::vector<ReactTestApp::ComponentViewModel> m_components;
+        ReactInstance m_reactInstance;
```

Comment:
Nit: For class members, let's postfix `_` instead of prefixing `m_`:
```suggestion
        ReactInstance reactInstance_;
```